test/test-get-one-week-arrival.js

```diff
@@ -0,0 +1,63 @@
+const assert = require('assert');
+
+const request = require('supertest');
+const app = require('../server');
+
+const mongodbMemoryServer = require('./util/mongodb-in-memory');
+const mockData = require('./util/mock-data');
+const mongoClientStub = {};
+const proxyRequire = require('proxyquire');
+const mongodbMock = proxyRequire('../server', {
+  './src/data/mongo-server': mongoClientStub
+});
+
+const faker = require('faker');
+
+const checkinSchema = require('../src/data/schema/checkin-schema');
+const usersSchema = require('../src/data//schema/user-schema');
+
+const momentjs = require('moment');
+
+describe("Get one week arrival data", () => {
```

Comment:
We should add a test where we have no data and we expect a 200 status code and an empty array.